### ISSUE TITLE
爱可可微博热门分享(2.25)

### ISSUE THREAD

**Issue by fly51fly** [fly51fly/aicoco#85]
No 1. <a href="https://weibo.com/1402400261/O24XAB5vq">Spider：Rust写的超快爬虫，支持并发、流式、分布式、无头浏览器、HTTP代理、定期执行、订阅等</a>
No 2. <a href="https://weibo.com/1402400261/O2cHavuK9">用原生PyTorch加速生成式AI模型</a>
No 3. <a href="https://weibo.com/1402400261/O2awwh6yU">[LG]《How Transformers Learn Causal Structure with Gradient Descent》</a>
No 4. <a href="https://weibo.com/1402400261/O2cG3Cf9J">用监督学习模型预测时间序列数据：将时间序列数据特征化为经典机器学习模型的标准表格数据，用AutoML提高精度</a>
No 5. <a href="https://weibo.com/1402400261/O24VawJNK">出门问问序列猴子开源数据集：出门问问提供的超大规模语言模型，基于其通用的表示与推理能力，支持多轮交互，能够大幅度提高生产效率和数据处理能力，被广泛应用于问答系统、自然语言处理、机器翻译、文本摘要等领域</a>
No 6. <a href="https://weibo.com/1402400261/O1TqzhLIA">2024年LLM研究趋势：模型合并，专家混合，更小的LLM</a>
No 7. <a href="https://weibo.com/1402400261/O25t314uE">System Design：系统设计快讯</a>
No 8. <a href="https://weibo.com/1402400261/O2cOdu9ln">Gemini从历史图像中强行去除白人形象的做法，不是消除种族主义的好办法</a>
No 9. <a href="https://weibo.com/1402400261/O2aGygEaQ">[CL] Large Language Models as Zero-shot Dialogue State Tracker ......</a>
No 10. <a href="https://weibo.com/1402400261/O2dzsiC24">OasysDB：可嵌入、高效且易于使用的向量数据库，可作为库嵌入到AI应用中。采用Rust编写，使用Sled作为持久化存储引擎，将向量集合保存到磁盘</a>
No 11. <a href="https://weibo.com/1402400261/O2dhCqUub">OpenAI Sora相关资源列表</a>
No 12. <a href="https://weibo.com/1402400261/O2aIy4iAL">[CL] CodeMind: A Framework to Challenge Large Language Models ......</a>
No 13. <a href="https://weibo.com/1402400261/O25nGhvvU">fast-audiomentations：利用GPU加速进行高效音频增强的Python库，适用于高吞吐量的音频分析和机器学习应用</a>
No 14. <a href="https://weibo.com/1402400261/O25vT072c">几篇论文实现代码</a>
No 15. <a href="https://weibo.com/1402400261/O22v6tUnd">Mamba通俗解析</a>
No 16. <a href="https://weibo.com/1402400261/O2atRFzXf">早！[太阳] ​</a>
No 17. <a href="https://weibo.com/1402400261/O2aQhE9Oh">[CL] ∞Bench: Extending Long Context Evaluation Beyond 100K ......</a>
No 18. <a href="https://weibo.com/1402400261/O22BxoNAS">AI透明背景贴纸生成，基于Stickers SDXL lora和BRIA’s remove background tool</a>
No 19. <a href="https://weibo.com/1402400261/O2du3Bz5x">pixel-profile：利用GitHub数据生成像素艺术头像。借助Github Readme Stats、Resvg Js和Satori等工具，该项目不仅可以生成GitHub统计卡片，还支持自定义样式</a>
No 20. <a href="https://weibo.com/1402400261/O27UepJD7">晚安～ [月亮] ​</a>
No 21. <a href="https://weibo.com/1402400261/O2aNlDSPZ">[CV] A Touch, Vision, and Language Dataset for Multimodal Alignment ......</a>
No 22. <a href="https://weibo.com/1402400261/O22uqlixS">Matryoshka嵌入模型介绍</a>
No 23. <a href="https://weibo.com/1402400261/O2aw84uaq">揭示了线性Transformer在上下文学习中的能力，证明了它们能内部实现一种集成了动量和基于噪声水平自适应重缩放的复杂优化算法，这种算法能在受噪声影响的数据上胜过多个基准算法，显示出线性Transformer在发现高级优化策略上的潜力。//[LG]《Linear Transformers are Versatile In-Context Learners》</a>
No 24. <a href="https://weibo.com/1402400261/O2dqffezu">KITTI Motion Compensation Library：KITTI运动补偿库，解决了KITTI激光雷达滚动快门效应带来的畸变问题</a>
No 25. <a href="https://weibo.com/1402400261/O2dameXHx">trzsz-ssh ( tssh )：SSH客户端的替代品，除了提供标准功能外，还增加了诸如登录提示、trzsz(trz / tsz)、批量登录、记住密码、zmodem(rz / sz)等实用功能</a>
No 26. <a href="https://weibo.com/1402400261/O2aGqyrbO">[CL]《Soft Self-Consistency Improves Language Model Agents》</a>
No 27. <a href="https://weibo.com/1402400261/O1O1Fd62U">'收集C/C++新手学习的入门项目，整理收纳开发者开源的小项目、工具、框架、游戏等，视频，书籍，面试题/算法题，技术文章。' ......</a>
No 28. <a href="https://weibo.com/1402400261/O1O02hjTG">gemma.cpp：轻量、独立的 C++ 推理引擎，用于 Google Gemma 基础模型</a>
No 29. <a href="https://weibo.com/1402400261/O24ZuE10m">micropip：Pyodide的轻量Python包安装器</a>
No 30. <a href="https://weibo.com/1402400261/O2aKqDrEb">[CL] Improving Language Understanding from Screenshots 网页链接 ......</a>
No 31. <a href="https://weibo.com/1402400261/O2d3z2JFG">DebateTree：用于可视化辩论的语言链应用，灵感源自思维树推理</a>
No 32. <a href="https://weibo.com/1402400261/O1sA5rZKd">AutoPrompt：一个提示优化框架，旨在增强和完善提示，以适应实际应用场景。该框架自动生成高质量、详细的提示，通过迭代构建具有挑战性的边界案例数据集，并进行优化。AutoPrompt不仅减少了提示工程的手动工作量，还有效解决了常见问题，如提示敏感性和内在提示模糊性</a>
No 33. <a href="https://weibo.com/1402400261/O1AjxxAIj">Sora物理悖谬的几何解释</a>
No 34. <a href="https://weibo.com/1402400261/O2aW1hNWv">今日推介(第1326期)：多语言Transformer的内在语言研究、线性Transformer是万能的上下文学习器、Transformer如何用梯度下降法学习因果结构、Transformer语言适配器的隐性空间、用软自洽增强语言模型Agent ......</a>
No 35. <a href="https://weibo.com/1402400261/O2axgp4h2">[CL]《The Hidden Space of Transformer Language Adapters》</a>
No 36. <a href="https://weibo.com/1402400261/O1N4kbG0Q">Transformer Lab：大型语言模型实验室应用，支持以下特性：下载数百种流行模型；通过聊天或完成与模型交互；使用数据集库或您自己的数据对模型进行微调；评估模型；通过 REST API 访问所有功能；通过简单的跨平台 GUI 来完成以上所有操作</a>
No 37. <a href="https://weibo.com/1402400261/O2d6g7D8U">maily.to：强大的编辑器，用于创建美观、预先设计、适用于移动设备的电子邮件</a>
No 38. <a href="https://weibo.com/1402400261/O2aw31ZE3">[LG]《Linear Transformers are Versatile In-Context Learners》</a>
No 39. <a href="https://weibo.com/1402400261/O1O27A3jk">WikiLLM：利用算法如MEMIT将事实注入LLM中，通过用户互动实时更新神经知识库</a>
No 40. <a href="https://weibo.com/1402400261/O1MNcsgSK">libLLM: 专为在普通个人电脑和移动设备上高效推理大型语言模型(LLM)而设计的开源项目</a>
No 41. <a href="https://weibo.com/1402400261/O1V2XoDKQ">Jax高性能LLM课程，旨在从头开始构建一个全尺度、高性能LLM，使用Jax进行训练和推理，涵盖了roofline分析、编译、分片、性能分析等内容</a>
No 42. <a href="https://weibo.com/1402400261/O1Tnvzijh">Stable Video视频生成正式上线</a>
No 43. <a href="https://weibo.com/1402400261/NzMmof7B3">2023年终大放送 之 TOP50开源项目榜：No. 1 ChatGPT 中文指南，指令指南，精选资源清单，更好的使用 ......</a>
No 44. <a href="https://weibo.com/1402400261/O1Ann0Ceg">GPT-4揭秘(外部视角)</a>
No 45. <a href="https://weibo.com/1402400261/O2awBqcce">通过引入一个上下文学习任务，并证明了Transformer模型能通过梯度下降在注意力层编码潜在因果结构，尤其是在数据的因果图为树状结构时，提出了Transformer学习过程中的理论框架和实证结果，创新地揭示了Transformer如何内化序列数据的因果依赖性。//[LG]《How Transformers Learn Causal Structure with Gradient Descent》</a>
No 46. <a href="https://weibo.com/1402400261/O223jnzi0">[CL] Cleaner Pretraining Corpus Curation with Neural Web Scraping ......</a>
No 47. <a href="https://weibo.com/1402400261/O1X9Ys5E6">'paper-ai - 搜索真实文献并生成引用真实文献的AI论文' GitHub: github.com/14790897/paper-ai ......</a>
No 48. <a href="https://weibo.com/1402400261/O22FomtnA">用AI提前7年预测阿尔茨海默病</a>
No 50. <a href="https://weibo.com/1402400261/O2268xLIS">[CV] Cameras as Rays: Pose Estimation via Ray Diffusion 提出一种新的相机姿态估计方法，与传统全局参数化方法不同，将相机视为光线束并通过衍射光线来进行姿态估计。这一方法通过与空间图像特征紧密结合，提高了姿态估计的精度，并适用于集合级的变换器。为了解决稀疏视 ......</a>
No 49. <a href="https://weibo.com/1402400261/O1IDGCMrQ">[LG]《Neural Network Diffusion》</a>
